packages/docker/test/wait.coffee

```diff
@@ -15,15 +15,13 @@ describe 'docker.wait', ->
       @docker.rm
         container: 'nikita_test_wait'
         force: true
-      @docker.tools.service
+      await @docker.tools.service
         image: 'httpd'
         container: 'nikita_test_wait'
       setTimeout =>
         @docker.stop
           container: 'nikita_test_wait'
       , 50
```

Comment:
Removing the await really was not the solution I was expecting!!

Reply:
I've put that await by mistake the first time.
only this await should have been present like this now:
```
      await @docker.tools.service
        image: 'httpd'
        container: 'nikita_test_wait'
      setTimeout =>
        @docker.stop
          container: 'nikita_test_wait'
      , 50
```
setTimeout didn't wait until tools.service create and run a container, what made tests not passing.

Reply:
It seems like `@docker.stop` should be called after the assertion, no? I can't find a reason why it is inside a `setTimeout` when looking at the code.

Reply:
No, `@docker.wait` blocks the session untill a container stops. If you move `@docker.stop` after `@docker.wait` this test will never pass. Here, we stop container after a bit later after running `@docker.wait`

Reply:
ok